src/components/scenes/WalletListScene.tsx

```diff
@@ -89,31 +89,36 @@ export function WalletListScene(props: Props) {
   const handlePressDone = useHandler(() => setSorting(false))
 
   return (
-    <SceneWrapper hasTabs>
-      <WiredProgressBar />
-      {sorting && (
-        <View style={styles.headerContainer}>
-          <EdgeText style={styles.headerText}>{lstrings.title_wallets}</EdgeText>
-          <TouchableOpacity key="doneButton" style={styles.headerButtonsContainer} onPress={handlePressDone}>
-            <EdgeText style={styles.doneButton}>{lstrings.string_done_cap}</EdgeText>
-          </TouchableOpacity>
-        </View>
+    <SceneWrapper hasTabs hasNotifications>
+      {gap => (
+        <>
+          <WiredProgressBar />
+          {sorting && (
+            <View style={styles.headerContainer}>
+              <EdgeText style={styles.headerText}>{lstrings.title_wallets}</EdgeText>
+              <TouchableOpacity key="doneButton" style={styles.headerButtonsContainer} onPress={handlePressDone}>
+                <EdgeText style={styles.doneButton}>{lstrings.string_done_cap}</EdgeText>
+              </TouchableOpacity>
+            </View>
+          )}
+          <View style={styles.listStack}>
+            <CrossFade activeKey={sorting ? 'sortList' : 'fullList'}>
+              <WalletListSwipeable
+                key="fullList"
+                header={header}
+                footer={searching ? undefined : footer}
+                navigation={navigation}
+                paddingBottom={gap.bottom}
```

Comment:
`style={ marginBottom: -gap.bottom }`, but maybe we should just pass in a `bottomGap` property, and let the list itself do the right thing.

Reply:
No this is right, it should be positive. The notificationHeight is meant as an
contentContainerStyle paddingBottom (overscroll) - using it as a style padding causes a visual cutoff